src/main/java/com/minecolonies/coremod/colony/buildings/views/BuildingBuilderView.java

```diff
@@ -59,26 +59,15 @@ public void deserialize(@NotNull ByteBuf buf)
             //final ItemStack itemStack = ByteBufUtils.readItemStack(buf);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)